### PR TITLE
update docstrings in F1d driver

### PR DESCRIPTION
**Pull request by peendebak**
- Improve docs
- Change units argument into unit

@jenshnielsen @WilliamHPNielsen @Dominik-Vogel @Christian-Volk

### CODE REVIEW

File: qcodes/instrument_drivers/QuTech/F1d.py [hunk 1]
```diff
@@ -20,6 +20,13 @@ class F1d(Instrument):
             individual modules.
 
         module (int): module number as set on the hardware.
+
+    Parameter:
+        enable_remote: set module in remote mode, switched on the front panel then deactivated
```
Comment (astafan8) [inline qcodes/instrument_drivers/QuTech/F1d.py]: perhaps you would want to add these lines to the `docstring` argument of the `add_parameter` function? :)

File: qcodes/instrument_drivers/QuTech/F1d.py [hunk 2]
```diff
@@ -49,17 +56,21 @@ def __init__(self, name, spi_rack, module, **kwargs):
...
-                           set_cmd=self.f1d.get_LO_level,
-                           units='dBm')
-
-        self.add_function('enable_remote', call_cmd=self.f1d.enable_remote)
-        self.add_function('clear_rf_clip', call_cmd=self.f1d.clear_rf_clip)
-        self.add_function('is_rf_clipped', call_cmd=self.f1d.rf_clipped)
+                           get_cmd=self.f1d.get_LO_level,
+                           unit='dBm')
+
+        self.add_parameter('enable_remote',
```
Comment (astafan8) [inline qcodes/instrument_drivers/QuTech/F1d.py]: why does this parameter not have a "get" command?

File: qcodes/instrument_drivers/QuTech/F1d.py (hunk 1 above)
Comment (astafan8) [inline qcodes/instrument_drivers/QuTech/F1d.py]: what does this line mean? when I read it, I do not understand: front panel? deactivated "then"?

File: qcodes/instrument_drivers/QuTech/F1d.py (hunk 2 above)
Comment (peendebak) [reply qcodes/instrument_drivers/QuTech/F1d.py]: @astafan8 This is due to hardware limitations

Comment (astafan8) [reply qcodes/instrument_drivers/QuTech/F1d.py]: how do you then know if the "remote" is enabled or not?

File: qcodes/instrument_drivers/QuTech/F1d.py [hunk 3]
```diff
@@ -66,7 +66,7 @@ def __init__(self, name, spi_rack, module, **kwargs):
 
         self.add_parameter('enable_remote',
                            label='Enable remote',
-                           set_cmd=self.f1d.enable_remote)
+                           set_cmd=self.f1d.enable_remote, docstring='getting the remote status is not possible')
```
Comment (astafan8) [inline qcodes/instrument_drivers/QuTech/F1d.py]: there is no need for this explanation, just add `get_cmd=False` :)

**Comment by jenshnielsen** [comment]
No one in Copenhagen have access to the QuTech Instruments. Modulo comments from @astafan8 it's fine with me to merge but I cannot test it @sohailc Do you have access to the F1D?

**Comment by astafan8** [comment]
@peendebak I see that you responded to most of the comments. Shall I merge it?

**Comment by eendebakpt** [comment]
Yes. The other drivers need some more time, this one is good

Op vr 27 jul. 2018 10:38 schreef Mikhail Astafev <notifications@github.com>:

**Pull request merged**